src/MTC4PF/include/MTC4PF.h

```diff
@@ -16,18 +16,20 @@
 // ***********************
 
 // MotorShieldType represents the type of the motor shield that is attached to the controller
-// L298N and L9110 are motorshields that are physically connected to the controller
+// L298N, L9110 and RED_L9110 are motorshields that are physically connected to the controller
 // LEGO_IR_8884 is connected via an infrared LED that must be wired to the controller
 enum struct MotorShieldType {
     NONE,
     L298N,
     L9110,
-    LEGO_IR_8884
+    LEGO_IR_8884,
+    RED_L9110
 };
 
 // Min and max useful power for Arduino based motor shields
 const int MIN_ARDUINO_POWER = 0;  // default minimum useful arduino power. May be overwritten in motor shield configuration
-const int MAX_ARDUINO_POWER = 1023; // default maximum arduino power. May be overwritten in motor shield configuration
+const int MAX_ARDUINO_POWER = 255; // default maximum arduino power. May be overwritten in motor shield configuration
+const int ANALOG_WRITE_RANGE = 255;
```

Comment:
I added this because the red shield needs this for power level calculation. Power level is ANALOG_WRITE_RANGE - powerLevel.
This is a breaking change for configurations which have [min|max]ArduinoPower set based on the assumption that ANALOG_WRITE_RANGE is 1023. On the other hand It might be worth thinking about this change, because it's the new default anyways.
To prevent issues on the long run I added explicit analog write range setting in setup.
Users with custom [min|max]ArduinoPower just need to adapt their configs accordingly.